schedule/yast/textmode/textmode@ppc64le.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: textmode@ppc64le
```

Comment:
shouldn't have some validation after first_boot module?
We still use PowerKVM, so I guess the file should reflect spvm or pvm_hmc. If that was the intention, please, check for other examples, maybe we can find better name.
Alternative I believe that this one could be unified via BACKEND variable, wdyt?

Reply:
there are for the `ppc64le-kvm-sle12`. `schedule/yast/textmode/textmode@ppc64le.yaml` is scheduled with ppc64le-hmc and ppc64le-spvm. the schedule for them are as they are presented at Functional group

Reply:
I'm not sure I get your answer, I will rephrase it:
- The name of the file does not reflect that this schedule is not valid for all ppc64le scenarios. We need a better name.
- Is it possible to unify via BACKEND variable?

Reply:
I will point you to https://gitlab.suse.de/qsf-y/qa-sle-functional-y/-/merge_requests/265. The name can be changed to something `textmode@ppc64le_hmc_spvm.yaml`. Does this work for you?

Reply:
yes, makes sense

Reply:
for this two flavor of ppc we should add the same modules to validate after first boot than in the other architectures (at least the fail and we could consider to do in a different ticket). Could you test that?

Reply:
The schedulers are the same as they appear in Functional group https://openqa.suse.de/tests/4556655 https://openqa.suse.de/tests/4556659. They fail. at least https://openqa.suse.de/tests/4565553. so yes if you want we can have a new ticket for this coverage.